configure.in

```diff
@@ -1402,7 +1402,7 @@ Use --without-zlib to disable zlib support.])])
 fi
 
 if test "$with_zstd" = yes; then
-  AC_CHECK_LIB(zstd, ZSTD_compressCCtx, [],
+  AC_CHECK_LIB(:libzstd.a, ZSTD_compressCCtx, [],
```

Comment:
I think not all changes in configure match changes in configure.in

Reply:
Can you point to that places?

Reply:
I just run `autoreconf -vif`

Reply:
Still can't see, can you show?